src/qe/mbp/mbp_qel.h

```diff
@@ -0,0 +1,41 @@
+/*++
+
+  Module Name:
+
+    mbp_qel.h
+
+Abstract:
+
+    Model Based Projection based on term graph
+
+Author:
+
+    Hari Govind V K (hgvk94) 2022-07-12
+
+Revision History:
+
+
+--*/
+
+#pragma once
+
+#include "ast/ast.h"
+#include "model/model.h"
+#include "util/params.h"
+
```

Comment:
use C++ format for class declaration. See other headers where "public" keyword appears